packages/components/src/components/post-breadcrumb/post-breadcrumb.tsx

```diff
@@ -29,6 +29,7 @@ export class PostBreadcrumb {
 
   @Watch('homeUrl')
   validateUrl() {
+    checkNonEmpty(this, 'homeUrl');
```

Comment:
The `homeUrl` is mandatory here so the `checkNonEmpty()` validator is appropriate. However, to make sur the property is correctly marked as mandatory in the docs it should be declared with the `!:` notation on line 16.